makefile

```diff
@@ -25,6 +25,9 @@ stop: stop-supabase ## stop the stack locally
 build: ## build the app
 	npm run build
 
+build-lib: ## build the library
+	npm run build-lib
+
```

Comment:
Shouldn't this be added to the `make build` target? I'm worried we risk forgetting to run `make build-lib`  next time we release the package to npm :grimacing:  .

Reply:
Actually both builds are separate: one is used to deploy the CRM app, the other is used to build the lib for npm. So keeping the 2 targets separate is a good thing.